cluster/ubuntu/util.sh

```diff
@@ -450,7 +451,8 @@ function provision-node() {
       '${KUBELET_CONFIG}'
     create-kube-proxy-opts \
       '${1#*@}' \
-      '${MASTER_IP}' 
+      '${MASTER_IP}' \
+      '${KUBE_PROXY_EXTRA_OPTS}'
```

Comment:
This command line wants the KUBE_PROXY_EXTRA_OPTS in one quoted string argument.  It is used in new line 252 without quoting, so that spaces here make for multiple arguments there.